generators/app/templates/tests/features/test.config.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
```

Comment:
I know lims-app does this
https://github.cerner.com/GenomicSolutions/lims-app/blob/master/webpack.base.config.js
https://github.cerner.com/GenomicSolutions/lims-app/blob/master/webpack.config.js
https://github.cerner.com/GenomicSolutions/lims-app/blob/master/webpack.hot.config.js